api/enrich.py

```diff
@@ -0,0 +1,196 @@
+from functools import partial
+from uuid import uuid4
+
+import requests
+from datetime import datetime, timedelta
+from flask import Blueprint, current_app
+
+from api.schemas import ObservableSchema
+from api.utils import get_json, jsonify_data, jsonify_errors
+
+enrich_api = Blueprint('enrich', __name__)
+
+
+get_observables = partial(get_json, schema=ObservableSchema(many=True))
+
+
+def get_judgement(observable_value, observable_type,
+                  disposition, valid_time):
+
+    source_uri = current_app.config['API_URL'] \
+                 + current_app.config['API_SOURCE'].format(
+        observable=observable_value)
+
+    return {
+        'id': f'transient:{uuid4()}',
+        'observable': {'value': observable_value, 'type': observable_type},
+        'disposition': disposition,
+        'disposition_name': 'Malicious',
```

Comment:
`disposition_name` depends on `disposition`, so you should use `current_app.config['DISPOSITIONS']` to map a `disposition` to its corresponding `disposition_name`.